package.json

```diff
@@ -65,6 +66,7 @@
     "unist-util-map": "^1.0.4",
     "unist-util-select": "^2.0.0",
     "uri-js": "^4.2.2",
+    "url": "^0.11.0",
```

Comment:
Try to use `uri-js` instead of `url`.

Reply:
I chose `url` because `image-handler.js` was already using it.

Reply:
Fixed in https://github.com/adobe/hypermedia-pipeline/pull/120/commits/db14b2c6e01f917f1d116c4e419980003569abd2